components/diagnosis-suggestion/api/pom.xml

```diff
@@ -29,7 +29,7 @@
   <name>PhenoTips - Diagnosis Suggestion Java Code</name>
 
   <properties>
-    <coverage.instructionRatio>0.81</coverage.instructionRatio>
+    <coverage.instructionRatio>0.78</coverage.instructionRatio>
```

Comment:
It's rarely OK to decrease the coverage ratio, you should add a new test instead.